src/NexusMods.App/Startup.cs

```diff
@@ -100,7 +100,8 @@ public static AppBuilder BuildAvaloniaApp(IServiceProvider serviceProvider)
             .UsePlatformDetect()
             .LogToTrace()
             .UseR3()
-            .UseReactiveUI();
+            .UseReactiveUI()
+            .With(new SkiaOptions { UseOpacitySaveLayer = true });
```

Comment:
Please annotate why this is needed.

Reply:
Opacity doesn't work on SVGs without this (see https://github.com/AvaloniaUI/Avalonia/pull/9964). It's needed for when we want to disable\fade SVG icons.

Reply:
Want me to add as a comment in the code ?

Reply:
Please add a comment in the code, linking to the relevant information.